frontend/src/app/projects/project-detail/model-overview/model-overview.component.ts

```diff
@@ -38,4 +43,23 @@ export class ModelOverviewComponent implements OnInit {
     }
     return 'Unset';
   }
+
+  openPipelineDialog(model: Model): void {
+    this.dialog.open(TriggerPipelineComponent, {
+      data: { project: this.project, model: model },
+    });
+  }
+
+  getPrimaryGitModelURL(model: Model): string {
```

Comment:
This function is called on every dom event, so it shouldn’t be the one that display the error.

Reply:
It’s the source of the error noticed there: https://github.com/DSD-DBS/capella-collab-manager/pull/349#issuecomment-1311477993

Reply:
Absolutely. I thought I fixed it already, but I'll have to check what happened with the commit.

Reply:
Ah, I fixed it as part of https://github.com/DSD-DBS/capella-collab-manager/pull/359. Should be fixed now after I merged the PR into this one :)